cosmos/models/Stage.py

```diff
@@ -14,7 +14,9 @@
 
 @signal_stage_status_change.connect
 def stage_status_changed(stage):
-    stage.log.info('%s %s (%s/%s Tasks were successful)' % (stage, stage.status, sum(t.successful for t in stage.tasks), len(stage.tasks)))
```

Comment:
There's a bug in a recent release of SqlAlchemyUtils which were causing defaults not to be set (so t.successful was None instead of False).  Did you create a new VE special for Cosmos?  It's fine in a PIPE environment which has SqlAlchemyUtils pinned to a working version, but a new Cosmos VE might break.  This is my fault for not merging master into develop which I just did.  Merge in develop and revert this line and you should be fine.